README.md

```diff
@@ -283,6 +151,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
 
```

Comment:
- [ ] Please your license link in your readme file is a broken link so make sure that you fix it :pray: